.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,68 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+
+
+jobs:
+  integration:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # gets full repo history
+          fetch-depth: 0
```

Comment:
I don't think we'll need fetch-depth anymore (that was a pants thing)

Reply:
Thanks!!!!!

Reply:
I'll leave it there for the moment until we remove pants from the actions (how close are we already?). Otherwise the tests won't work.

Reply:
Removed.